src/sass/_pattern_navigation.scss

```diff
@@ -1,35 +1,34 @@
-@use "sass:map";
 @import "./breakpoints";
 
 .p-panel__logo {
   margin-top: 0;
 
   .p-panel__logo-image {
-    height: 48px;
+    height: 36px;
+    margin-top: -3px;
     max-width: inherit;
   }
 
   .logo-text {
     color: #fff;
-    font-size: #{map.get($font-sizes, h4)}rem;
-    left: 54px;
+    left: 47px;
     position: absolute;
-    top: 21px;
+    top: 5px;
   }
 
   @include mobile {
     .logo-text {
-      left: 46px;
+      left: 38px;
     }
   }
 
   @include extra-large {
     .p-panel__logo-image {
-      height: 54px;
+      height: 40px;
     }
 
     .logo-text {
-      left: 61px;
+      left: 52px;
```

Comment:
I see the logo text "jumping up" when passing the `x-large` breakpoint:

![Peek 2023-03-28 12-23](https://user-images.githubusercontent.com/56583786/228207613-98ab58a3-75ab-4b34-aa82-e61a2e786c3e.gif)

Maybe it needs some `top` adjustment too? I think it's ugly to see it jump, even though the title of the instance list does exactly the same...

Reply:
Good catch! the top spacing of the logo text should be `.25rem` instead of `5px`.

Reply:
I don't see any change in the `x-large` view, it's still jumping just as shown in the screen capture. But maybe this is expected? I see that it happens with the title as well, it's just that it looks super-ugly to me...

Reply:
Before it was not in line with the other headings, now it is. As for the jumping, I think that is intended for larger screens, if understand your concern correctly, even the [vanilla example](https://vanillaframework.io/docs/examples/layouts/application/JAAS) has the same behaviour.